guides/source/webpacker.md

```diff
@@ -177,18 +179,15 @@ As of Webpacker version 5, Webpacker is not "engine-aware," which means Webpacke
 
 Gem authors of Rails engines who wish to support consumers using Webpacker are encouraged to distribute frontend assets as an NPM package in addition to the gem itself and provide instructions (or an installer) to demonstrate how host apps should integrate. A good example of this approach is [Alchemy CMS](https://github.com/AlchemyCMS/alchemy_cms).
 
-### Hot module replacement
+### Hot Module Replacement (HMR)
```

Comment:
The mention of `HMR` below confused me at first, so I thought this may clarify it.